services/121-service/src/financial-service-providers/repositories/financial-service-provider-question.repository.ts

```diff
@@ -0,0 +1,24 @@
+import { FinancialServiceProviderName } from '@121-service/src/financial-service-providers/enum/financial-service-provider-name.enum';
+import { FspQuestionEntity } from '@121-service/src/financial-service-providers/fsp-question.entity';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Equal, Repository } from 'typeorm';
+
+export class FinancialServiceProviderQuestionRepository extends Repository<FspQuestionEntity> {
+  constructor(
+    @InjectRepository(FspQuestionEntity)
+    private financialServiceProviderQuestionRepository: Repository<FspQuestionEntity>,
```

Comment:
@PeterSmallenbroek Suggestion to call this variable e.g. "baseRepository" as now this variable now looks like an object of the class it is defined in, which may be confusing to the reader.